.eslintrc.json

```diff
@@ -173,7 +173,7 @@
     "no-restricted-imports": [
       "error",
       {
-        "paths": ["lodash"]
+        "paths": ["lodash", "@material-ui/icons/Delete"]
```

Comment:
I don't understand, why should we include it here but not other icons?

Reply:
Because it's easy to miss as it's both in macaw and mui. If we stumble upon more icons that are in both packages we'd include them here, though I don't have any candidate for this now. Ideally we'd get rid of `@material-ui/icons` but it's not happening right now.